asf_tools/raster.py

```diff
@@ -0,0 +1,29 @@
+import warnings
+import numpy as np
+
+
+def convert_scale(array: np.ndarray, in_scale: str, out_scale: str) -> np.ndarray:
```

Comment:
we're supporting python 3.8+, so we could use `in_scale: Literal['db', 'amplitude', 'power']` here